docs/changes.rst

```diff
@@ -58,6 +84,10 @@ Released: not yet
 * Docs: Switched to using the sphinx_rtd_scheme for the HTML docs
   (See issue #2367).
 
+* Replaced pywbem's own NocaseDict with NocaseDict from the nocasedict package
```

Comment:
Should this not be in the enhancements section or at least not in bug fixes.

Reply:
It is currently in the Cleanup section. I did not see much of a functional enhancement that would qualify it for the enhancement section.

Reply:
Ok. done.